recipesearch/src/Recipes/SearchResults/SearchResults.js

```diff
@@ -1,17 +1,21 @@
 /* eslint-disable jsx-a11y/img-redundant-alt */
 import React, { Component } from "react";
 import { connect } from "react-redux";
+import { searchIngredientsRecipesAction } from "../SearchIngredients/action";
+import { Link } from 'react-router-dom';
 
 export class SearchResults extends Component {
   render() {
     const { searchResults } = this.props;
+    console.log(searchResults)
     return (
       <div>
         {searchResults && searchResults.map((searchResult) => (
-          <div key={searchResult.idMeal}>
+          <div  >
             <h1>{searchResult.strMeal}</h1>
             <p>{searchResult.strInstructions}</p>
             <img src={searchResult.strMealThumb} alt= {`Picture for ${searchResult.strMeal}`}/>
+            <Link to="/ingredients" onClick={() => {this.props.searchIngredientsRecipesAction(searchResult.idMeal)}}>View Ingredients</Link>
```

Comment:
I think it is possible but the issue arises when accessing the Id since it is acquired from a specific meal which can be attained from looping through all meals. This therefore makes it difficult to access the Id

Reply:
Yes it is possible